public_html/wp-content/mu-plugins/blocks/assets/src/blocks-store.js

```diff
@@ -0,0 +1,160 @@
+/**
+ * External dependencies
+ */
+import { intersection, orderBy, split } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+const { registerStore, select, dispatch } = wp.data;
+const apiFetch                            = wp.apiFetch;
+
+/**
+ * Define store name.
+ */
+export const WC_BLOCKS_STORE = 'wordcamp';
+
+const MAX_POSTS = -1 ; // Its supported now.
+
+const DEFAULT_STATE = {};
+
+/**
+ * Helper method to apply filters. Can be moved to a seperate file.
+ * Don't use the output for this in a prop because it could cause perf issues,
+ * since ref of returned object will always be different.
+ *
+ * @param { Array }  entities List of entities to filter against
+ * @param { Object } args     Arguments for filter function
+ */
+export const filterEntities = ( entities, args ) => {
+	if ( ! Array.isArray( entities ) ) {
+		return entities;
+	}
+
+	let result = [ ...entities ];
+
+	if ( args.hasOwnProperty( 'filter' ) && Array.isArray( args.filter ) ) {
+		args.filter.forEach( ( filterParams ) => {
+			let { fieldName, fieldValue } = filterParams;
+
+			if ( ! Array.isArray( fieldValue ) ) {
+				fieldValue = [ fieldValue ];
+			}
+
+			result = result.filter( ( entity ) => {
+				if ( ! entity.hasOwnProperty( fieldName ) ) {
+					return false;
+				}
+
+				let compareValue = Array.isArray( entity[ fieldName ] ) ? entity[ fieldName ] : [ entity[ fieldName ] ];
+
+				return intersection( fieldValue, compareValue ).length > 0;
+			} );
+		} );
+	}
+
+	if ( args.hasOwnProperty( 'order' ) ) {
+		let [ orderby, order ] = split( args.order, '_', 2 );
```

Comment:
[SHOULD] I was confused by having this property be named `order`, since that is also the name of one of the two parameters that this splits into, that is used in WP queries. Thus, I would expect a property called `order` to only contain a string value of `ASC` or `DESC`.

I think it would be better and avoid more confusion to have this property be named `sort`, since that is the block attribute that the blocks are feeding into this anyway.

Reply:
Changed name to `sort`: 69b4687dc0343218bd3c8682aa1b1d71de5dde8d